crates/nu-command/src/filters/par_each.rs

```diff
@@ -298,6 +292,7 @@ impl Command for ParEach {
                 )
             }
         }
+        .and_then(|x| x.filter(|v| !v.is_nothing(), outer_ctrlc))
```

Comment:
Is this checking for `$nothing` to match the behavior of `each` or to effectively fuse the iterator?

Reply:
It's to match `each`. It's not a behavior change though. Previously, `par-each` would already discard `Nothing` like `each` does (and one of the existing examples checks for that). 

It was just implicit due to the `PipelineDataIterator` being flattened I think. So this just keeps existing behavior.

Reply:
Ah, all good!